nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/opvalidation/ShapeOpValidation.java

```diff
@@ -2525,4 +2527,45 @@ public void testReshapeZeros(){
             assertArrayEquals(exp, out.shape());
         }
     }
+
+
+    @Test
+    public void testMergeMaxIndex() {
+
+        Nd4j.getRandom().setSeed(12345);
+        SameDiff sd = SameDiff.create();
+        SDVariable inputX = sd.var(Nd4j.createFromArray(new float[] {1, 0, 0}));
+        SDVariable inputY = sd.var(Nd4j.createFromArray(new float[] {0, 1, 0}));
+        SDVariable inputZ = sd.var(Nd4j.createFromArray(new float[] {0, 0, 1}));
+        SDVariable out = new MergeMaxIndex(sd, inputX, inputY, inputZ).outputVariable();
+        INDArray expected = Nd4j.createFromArray(new long[]{0,1,2});
+        assertArrayEquals(expected.toIntVector(), out.eval().toIntVector());
+
+
+    }
+
+    @Test
+    public void testTriOp() {
+
+        SameDiff sd = SameDiff.create();
+        SDVariable out = new Tri(sd, 3, 5, 2).outputVariable();
+        INDArray expected = Nd4j.createFromArray(new long[][]{{1, 1, 1, 0, 0}, {1, 1, 1, 1, 0}, {1, 1, 1, 1, 1}});
+        assertArrayEquals(expected.toLongMatrix(), out.eval().toLongMatrix());
```

Comment:
For both testMergeMaxIndex and testTriOp - use OpValidation.
That way, the tests count towards test coverage (i.e., the output of OpValidationSuite will count them as being tested)